### PR TITLE
Python2.7 support

### PR DESCRIPTION
**Pull request by blag**
I depend on the [`monotonic` project](https://pypi.org/project/monotonic) instead of `time.monotopic`. According to its description, "On Python 3.3 or newer, monotonic will be an alias of time.monotonic from the standard library", so it is compatible with both Python 2.7 and Python 3.3+.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -93,6 +97,7 @@ def run_tests(self):
 tests_require = ["pytest", "pytest-cov", "pytest-timeout>=0.3"]
 install_requires = [
     "pathtools3>=0.2.0",
+    "monotonic>=1.5",
```
Comment (BoboTiG) [inline setup.py]: Could you make the requirement for Python 2 only? And add the `monotonic()` export into `compat.py` instead. Like that, on Python 3 there will not be a useless requirement to install.

File: src/watchdog/observers/inotify_c.py [hunk 2]
```diff
@@ -195,7 +195,10 @@ def __init__(self, path, recursive=False, event_mask=WATCHDOG_ALL_EVENTS):
         self._path = path
         self._event_mask = event_mask
         self._is_recursive = recursive
-        self._add_dir_watch(path, recursive, event_mask)
```
Comment (BoboTiG) [inline src/watchdog/observers/inotify_c.py]: What is the purpose of this change?

File: setup.py (hunk 1 above)
Comment (blag) [reply setup.py]: Sure, I can do that.

File: src/watchdog/observers/inotify_c.py (hunk 2 above)
Comment (blag) [reply src/watchdog/observers/inotify_c.py]: This change accidentally snuck in - it allows users to set watches on individual files. I'll open another PR for it so this PR stays clean.

File: setup.py (hunk 1 above)
Comment (blag) [reply setup.py]: Actually, `monotonic` is only used in `test_delayed_queue`, so I'll just move it into `tests_require`.

**Comment by blag** [comment]
@BoboTiG I have corrected all of the issues you pointed out. Please let me know if there's anything else I need to do. Thanks!

**Comment by BoboTiG** [comment]
I will check and merge tomorrow, thank you!
